aws/resource_aws_ecs_service.go

```diff
@@ -568,6 +577,22 @@ func resourceAwsEcsServiceUpdate(d *schema.ResourceData, meta interface{}) error
 		return err
 	}
 
+	if d.Get("wait_for_steady_state").(bool) {
+		log.Println("[INFO] Waiting for service to reach a steady state")
+
+		steadyStateConf := &resource.StateChangeConf{
+			Pending:    []string{"false"},
+			Target:     []string{"true"},
```

Comment:
Returning a `bool` state as a string feels a bit weird, but there's precedent:

https://github.com/terraform-providers/terraform-provider-aws/blob/490897d8f64608a336ab43564c5dfe681509b213/aws/resource_aws_network_interface.go#L205-L208

I considered returning the service state or "STABLE", but then we're relying on ECS never introducing another state.